tests/manifests/common-cluster-secondary.yaml

```diff
@@ -0,0 +1,128 @@
+# This is a template to generate the necessary RBAC to deploy a second cluster into a namespace
+# It assumes that common.yaml already ran
+#
+# Run me like:
+# NAMESPACE=rook-ceph-secondary envsubst < tests/manifests/common-cluster-secondary.yaml | kubectl create -f -
+---
+apiVersion: v1
```

Comment:
I'm fine using this as an official example. However, I'd like to keep the env var since it allows us to do smooth substitution instead of relying on `sed` (still possible though) which has been annoying to maintain across platforms.